flows/slurm/slurm_flows.py

```diff
@@ -1 +1,75 @@
-# TODO
+# TODO: Check pyslurm: https://github.com/PySlurm/pyslurm/tree/main
+import sys
+import tempfile
+
+import yaml
+from prefect import context, flow, get_run_logger
+from prefect.states import Failed
+from prefect.utilities.processutils import run_process
+
+from flows.slurm.schema import SlurmParams
+
+
+class Logger:
```

Comment:
Can this be refactored into something common across the flows?

Reply:
We will take another look at this through the general refactoring.